crates/polars-error/src/lib.rs

```diff
@@ -247,7 +247,13 @@ Alternatively, if the performance cost is acceptable, you could just set:
 on startup."#.trim_start())
     };
     (duplicate = $name:expr) => {
-        polars_err!(Duplicate: "column with name '{}' has more than one occurrences", $name)
+        polars_err!(
+            Duplicate: "column '{}' is duplicate.\n\n\
```

Comment:
I don't think we should have this message for all duplicate related errors. Only in `select` and `with_columns`.

Reply:
No problem--I think I fixed that with latest commit.